### ISSUE TITLE
add support new for dm7080

### ISSUE THREAD

**Issue by atvcaptain** [christophecvr/gstreamer1.0-plugin-multibox-dvbmediasink#17]
check this thanks

http://git.opendreambox.org/?p=gst-plugin-dvbmediasink.git;a=commit;h=9dc98551a8ff9f43a2817fae10b1f0ddaf98effb

**Comment by christophecvr** [comment]
Strictly speaking You just could add the caps, adapt pes header box related.
Question is (the new driver with ...). What is that with. Then we also have the enigma2 support , Unable to see what needs to be add there since they went closed source.

**Comment by atvcaptain** [comment]
yes only add the caps, all other include in the drivers and only news 7435 chip have this dm is the first box, a new mutant 3400 with same chip is coming at end the year  , i think we can enable like other options with switch like this: --with-SPARK, --with-VP6, --with-VP8

**Comment by christophecvr** [comment]
If You test with You're images to modify  the sink.  And can get it working.
A patch specific box related whict procedures and or caps ad's change and or limit always welcome.

Just make it box specific and add the it with a option which can be set by the dvbmediaconfig
